cbv3_django_prototype/cbv3_django_prototype/resources/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import include, path
+from rest_framework import routers
+from . import views
+
+router = routers.DefaultRouter()
+router.register(r'resources', views.ResourceListView, basename='resources')
+
+urlpatterns = [
+    path('', include(router.urls)),
+    path('api-auth/', include('rest_framework.urls', namespace='rest_framework'))
+    #path('<int:id>/', views.DetailResource.as_view()),
```

Comment:
Lets remove commented code. We can add it when it is relevant.

Reply:
Yup.  Will remove on next revision.